pkg/api/config/config.go

```diff
@@ -512,6 +512,10 @@ func (c *Config) IsSyncEnabled() bool {
 	return c.Extensions != nil && c.Extensions.Sync != nil && *c.Extensions.Sync.Enable
 }
 
+func (c *Config) IsCompatEnabled() bool {
+	return len(c.HTTP.Compat) > 0
```

Comment:
Can we get coverage for this? How come this is not covered already, I would expect it to be triggered in the existing sync tests.